CTP/shell/src/com/navercorp/cubridqa/shell/main/Context.java

```diff
@@ -236,7 +235,41 @@ public String getTestCaseWorkspace() {
 			return ws;
 		}
 	}
-
+	
+	public String getTestPlatform(){
+		return getProperty("main.testing.platform", "linux");
+	}
+	
+	public String getTestCategory()
+	{
+		return getProperty("main.testing.category", "general");
+	}
+	
+	public boolean needCleanTestCase()
+	{
+		return this.cleanTestCase;
+	}
+	
+	public String getTestCaseTimeout()
+	{
+		return getProperty("main.testcase.timeout", "-1");
+	}
+	
+	public boolean needEnableMonitorTrace()
+	{
+		return getProperty("main.monitor.enable_tracing").equalsIgnoreCase("true");
```

Comment:
Provide a default value (false) for main.monitor.enable_tracing.

As is:
return getProperty("main.monitor.enable_tracing").equalsIgnoreCase("true");

To be:
return com.navercorp.cubridqa.common.CommonUtils.convertBoolean(getProperty("main.monitor.enable_tracing", "false"));